img/src/BlobClustering.cxx

```diff
@@ -117,26 +119,84 @@ intern_one(cluster_indexed_graph_t& grind,
         return end;
     }
 
-    if (rel_time_diff(*beg, *next) > spans) {
-        return next;
+    if (policy != "simple" and policy != "uboone") {
+        THROW(ValueError() << errmsg{String::format("policy %s not implemented!", policy)});
```

Comment:
Not a big deal but two things here:

- main thing: better to check for legal policy in `configure()` and throw early there instead of later during run time.
- really minor: better if `m_policy` is an `enum` set from string inside of `configure()`